### PR TITLE
New Entry: General Endianness

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

From Target Keyword List

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/general/concepts/endianness/endianness.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  - 'Code Foundations'
+Tags:
+  - 'Numbers'
+  - 'Memory'
+CatalogContent:
+  - 'paths/code-foundations'
+  - 'paths/computer-science'
+---
+
+Endianness is a description of the order bytes are stored in memory for multi-byte values. The options are generally described as big-endian or little-endian. A big-endian system stores the most significant byte at the smallest memory address and the least significant byte at the largest. A little-endian system stores the least significant byte at the smallest memory address and the most significant byte at the largest.
```
Comment (Dusch4593) [inline content/general/concepts/endianness/endianness.md:15]:
```suggestion
**Endianness** describes the order in which bytes are stored in memory for multi-byte values. The options are generally described as big-endian or little-endian. A big-endian system stores the most significant byte at the smallest memory address and the least significant byte at the largest. A little-endian system stores the least significant byte at the smallest memory address and the most significant byte at the largest.
```

File: content/general/concepts/endianness/endianness.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Example
+
+To store the 32-bit integer `0x0A0B0C0D` it is broken down into four bytes `0x0A`,`0x0B`,`0x0C`,`0x0D`. The endianness of the computer system defines in what order these bytes are stored in memory. The following examples show it stored at memory address `0xFFF0` for both big- and little-endian systems.
```
Comment (Dusch4593) [inline content/general/concepts/endianness/endianness.md:19]:
```suggestion
To store the 32-bit integer `0x0A0B0C0D` it is broken down into four bytes:

- `0x0A`
- `0x0B`
- `0x0C`
- `0x0D`

The endianness of the computer system defines in what order these bytes are stored in memory. The following examples show it stored at memory address `0xFFF0` for both big- and little-endian systems:
```

File: content/general/concepts/endianness/endianness.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+Title: 'Endianness'
+Description: 'Endianness is a description of the order bytes are stored in memory for multi-byte values.'
```
Comment (Dusch4593) [inline content/general/concepts/endianness/endianness.md:3]:
```suggestion
Description: 'Endianness describes the order in which bytes are stored in memory for multi-byte values.'
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Just a few comments/suggestions from me 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, I implemented your suggestions.

**Review by Dusch4593 — APPROVED** [review]
@SSwiniarski LGTM for one more review 👍🏻

**Review by KTom101 — APPROVED** [review]
@SSwiniarski I don't see any edits that are needed. I've suggested a comma in one spot.

**Comment by SSwiniarski** [comment]
@KTom101, I inserted your comma.

**Pull request merged**